File: src/Entity/Participant.php
<?php

namespace App\Entity;

use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

/**
 * Participant
 *
 * @ORM\Table(name="participant", indexes={@ORM\Index(name="id_evenement", columns={"id_evenement"})})
 * @ORM\Entity
 */
class Participant
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_participant", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idParticipant;

    /**
     * @var int
     *
     * @ORM\Column(name="id_user", type="integer", nullable=false)
     */
    private $idUser;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_part", type="date", nullable=false)
     */
    private $datePart;

    /**
     * @var \Evenement
     *
     * @ORM\ManyToOne(targetEntity="Evenement")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_evenement", referencedColumnName="id_ev")
     * })
     */
    private $idEvenement;

    public function getIdParticipant(): ?int
    {
        return $this->idParticipant;
    }

    public function getIdUser(): ?int
    {
        return $this->idUser;
    }

    public function setIdUser(int $idUser): static
    {
        $this->idUser = $idUser;

        return $this;
    }

    public function getDatePart(): ?\DateTimeInterface
    {
        return $this->datePart;
    }

    public function setDatePart(\DateTimeInterface $datePart): static
    {
        $this->datePart = $datePart;

        return $this;
    }

    public function getIdEvenement(): ?Evenement
    {
        return $this->idEvenement;
    }

    public function setIdEvenement(?Evenement $idEvenement): static
    {
        $this->idEvenement = $idEvenement;

        return $this;
    }


}

File: src/main/java/entities/participant.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Date;

/**
 *
 * @author asus
 */
public class participant extends evenement {
    private int id_participant;        
private Date date_part;    
private int id_user;
public int id_evenement; 
public evenement evenement;

    public participant() {
    }

    public participant(int id_participant, Date date_participant, int id_user, int id_evenement) {
        this.id_participant = id_participant;
        this.date_part = date_participant;
        this.id_user = id_user;
        this.id_evenement = id_evenement;
    }
    public participant(Date date_participant, int id_user, int id_evenement) {
        this.date_part = date_participant;
        this.id_user = id_user;
        this.id_evenement = id_evenement;
    }

    public participant(int id_participant, Date date_participant, int id_user, int id_evenement, evenement evenement) {
        this.id_participant = id_participant;
        this.date_part = date_participant;
        this.id_user = id_user;
        this.id_evenement = id_evenement;
        this.evenement = evenement;
    }

    public int getId_participant() {
        return id_participant;
    }

    public Date getDate_part() {
        return date_part;
    }

    public int getId_user() {
        return id_user;
    }

    public int getId_evenement() {
        return id_evenement;
    }

    public evenement getevenement() {
        return evenement;
    }

    public void setId_participant(int id_participant) {
        this.id_participant = id_participant;
    }

    public void setDate_part(Date date_participant) {
        this.date_part = date_participant;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public void setId_evenement(int id_evenement) {
        this.id_evenement = id_evenement;
    }

    public void setEvenement(evenement evenement) {
        this.evenement = evenement;
    }

    @Override
    public String toString() {
        return "participant{" + "id_participant=" + id_participant + ", date_participant=" + date_part + ", id_user=" + id_user + ", id_evenement=" + id_evenement +  '}';
    }
    
    
    




}
